yaru-theme: 20.10.1 -> 20.10.6.1

amaxine opened the pull request:
###### Motivation for this change
Version bump. Verified as working through use.

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

Commit:
yaru-theme: 20.10.1 -> 20.10.6.1

maxeaubrey commented:
Result of `nixpkgs-review pr 114289` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>yaru-theme</li>
  </ul>
</details>

r-rmcgibbo commented:
Result of `nixpkgs-review pr 114289` at 4a6e488c run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/7c18b2b852f70f1549b937092d84a98d">yaru-theme</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl3` is a deprecated license, check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/data/themes/yaru/default.nix:28:5:
```
   |
28 |     license = with licenses; [ cc-by-sa-40 gpl3 ];
   |     ^
```
</li>
  </ul>
</details>

Commit:
yaru-theme: update licenses

based on https://github.com/ubuntu/yaru/blob/1b74404c407ebf5a8487e9f138173701af8cb3e0/debian/copyright

jD91mZM2 reviewed (approved):
Looks good to me!
